challenges/sre/challenge.md

```diff
@@ -166,15 +171,12 @@ but, at a minimum, please include the following:
 
 ### Automation
 
-* Ability to build the server in a Docker container
-* Ability to run the server (inside and outside Kubernetes)
-* Ability to execute integration tests against the local Kubernetes cluster
-* Produce production ready releases (binaries and docker image)
+* Ability to build and deploy all artifacts to a Kubernetes cluster using make
+* Ability to execute integration tests against the local Kubernetes cluster using make
```

Comment:
```suggestion
* Write a Dockerfile to build an image for the server
* Ability to build and deploy all artifacts to a Kubernetes cluster using make
* Ability to execute integration tests against the local Kubernetes cluster using make
```

Noticed this one was consistent in L3 and below but didn't carry over.

Reply:
I left this one out at the later levels in case they want to use ko or similar. They need to build an image somehow, so it seemed unnecessary to require a Dockerfile specifically

Reply:
Gotcha that's fine 👍